### PR TITLE
task/WG-161-Kube-Cleanup

### PR DESCRIPTION
**Pull request by sophia-massie**
## Overview: ##
Kubernetes cleanup

## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## Related Jira tickets: ##

* [WG-161](https://tacc-main.atlassian.net/browse/WG-161)

## Summary of Changes: ##
- Removes kube directory 
- Adds updated deployment information in README

## Testing Steps: ##

None

## Notes: ##
Can add more to the README as we need

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,3 +82,18 @@ Run `npm run lint:css -- --fix` to fix css files.
...
+The deployment of images and updating of the services and applications is performed by a Jenkins
+workflow found [here](https://jenkins01.tacc.utexas.edu/view/Hazmapper+Geoapi/.)
+
+The images used in deployment are built automatically for the master branch using TravisCI and 
+pushed to Docker Hub (see https://hub.docker.com/r/taccaci/hazmapper).
+
+
+## Additional Information/Tips
+
+For additional information like access and troubleshooting see [geoapi's devops directory TODO: create devops/README.md](https://github.com/TACC-Cloud/geoapi/tree/master/devops).
```
Comment (nathanfranklin) [inline README.md:99]: Should we just point them to the other readme:

# Deployment

For deployment information, see https://github.com/TACC-Cloud/geoapi/blob/master/devops/README.md

Comment (nathanfranklin) [inline README.md:99]: Should we just point them to the other readme?


```suggestion

# Deployment

For deployment information, see https://github.com/TACC-Cloud/geoapi/blob/master/devops/README.md
```

Comment (sophia-massie) [reply README.md:99]: We can do that no problem! I was trying to emulate how it was done previously where it say the above pieces in both geoapi and hazmapper for thoroughness but also redundant. I see I didn't link directly to the devops/README and need to update that.

**Review by nathanfranklin — APPROVED** [review]
LGTM 👍

File: README.md [hunk 2]
```diff
@@ -82,3 +82,18 @@ Run `npm run lint:css -- --fix` to fix css files.
...
+## Images building + Jenkins
+
+The deployment of images and updating of the services and applications is performed by a Jenkins
+workflow found [here](https://jenkins01.tacc.utexas.edu/view/Hazmapper+Geoapi/.)
+
+The images used in deployment are built automatically for the master branch using TravisCI and 
+pushed to Docker Hub (see https://hub.docker.com/r/taccaci/hazmapper).
+
+
+## Additional Information/Tips
```
Comment (nathanfranklin) [inline README.md]:
```suggestion
## Configuration
```

**Review by taoteg — APPROVED** [review]
LGTM.

**Pull request merged**